index.js

```diff
@@ -279,10 +279,10 @@ class HtmlWebpackPlugin {
               return self.options.showErrors ? prettyError(err, compiler.context).toHtml() : 'ERROR';
             })
             .then(html => {
-              // Allow to use [contenthash] as placeholder for the html-webpack-plugin name
+              // Allow to use [templatehash] as placeholder for the html-webpack-plugin name
```

Comment:
This change seems unrelated?

Reply:
That is unrelated yes. But tests will fail without that.

Reply:
Ah makes sense. I guess this change fixes #1033.

Reply:
Yes. Just noticed there is also another PR #1034 does the same.

Reply:
Actually it's fine for me - maybe we could also replace `contenthash` to `templatehash` before we compile it.  
So users will be able to use `contenthash` and `templatehash`

Reply:
Cool. So that no breaking change will be introduced. 👍

Reply:
@jantimon checkout #1065